style.css

```diff
@@ -89,11 +95,66 @@ a:hover {
     color: var(---black);
 }
 
+/* hero section */
+
+.hero {
+    position: relative;
+    width: 100vw;
+    height: 82vh;
+    color: var(--white);
+    background-image: url(./assets/hero-image_large.jpeg);
+    background-size: cover;
+    background-position: center center;
+    background-repeat: no-repeat;
+    background-attachment: scroll;
+}
+
+
+.hero h1 {
+    font-size: 3em;
+    margin-bottom: 0.5em;
+}
+
+.hero h2 {
+    margin-bottom: 0.5em;
+    font-size: 1.5em;
+}
+
+.hero .search-input {
+    border-radius: 0.5em;
+}
+
+.hero a {
+    color: var(--gray-2);
+    text-decoration: none;
+}
+
+footer {
```

Comment:
Should be named `hero_footer`. Otherwise someone might think this is the footer of the page.